var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -66,10 +66,11 @@ class Openmpi(AutotoolsPackage):
     version('develop', branch='master')
 
     # Current
-    version('4.0.0', sha256='2f0b8a36cfeb7354b45dda3c5425ef8393c9b04115570b615213faaa3f97366b')  # libmpi.so.40.20.0
+    version('4.0.1', sha256='cce7b6d20522849301727f81282201d609553103ac0b09162cf28d102efb9709')  # libmpi.so.40.20.1
 
     # Still supported
-    version('3.1.3', preferred=True, sha256='8be04307c00f51401d3fb9d837321781ea7c79f2a5a4a2e5d4eaedc874087ab6')  # libmpi.so.40.10.3
```

Comment:
tbh, I would keep `3.1.3` a little longer as preferred until we tested default stacks for I/O libs (HDF5, NetCDF, ADIOS1/2, ...) and large applications work with it. I currently am busy with my thesis, but would otherwise assist in testing.